src/main.rs

```diff
@@ -6,26 +6,44 @@ mod models;
 mod rest;
 
 use crate::rest::RestServer;
-use std::env::args;
+use clap::Parser;
 use std::io::{Write, stdout};
 use std::process::exit;
 
+#[derive(Parser, Debug)]
+#[command(about = "rocd server")]
+struct CLI {
+    /// server host
+    #[arg(long, default_value_t = String::from("127.0.0.1"))]
```

Comment:
`default_value_t = String::from("xx")` => `default_value = "xx"`

Recommend way for strings

Reply:
ok